static/js/gymnasium.js

```diff
@@ -27,6 +27,95 @@ Gymnasium.prototype.setBackgroundColorOfElementFromImage = function (element, im
       $(element).css('background-color','rgba(' + r + ',' + g + ',' + b + ',' + a + ')');
     });
   })
-}
+};
+
+Gymnasium.prototype.RecordCourseEnrollment = function(email, courseid)
```

Comment:
These couple functions look blank... because they are, and they're yet unused.  Working on filling in the finer points of these implementations for an update to this PR in the coming days.

I swear I didn't put the blank functions in my commits, but here we are. ;)